src/main/java/org/jitsi/videobridge/IceDtlsTransportManager.java

```diff
@@ -169,37 +170,40 @@ public void onDtlsHandshakeComplete(Runnable handler)
 
     private Node createIncomingPipeline()
     {
+        // do we need a builder if we're using a single node?
         PipelineBuilder builder = new PipelineBuilder();
 
         DemuxerNode dtlsSrtpDemuxer = new DemuxerNode("DTLS/SRTP");
         // DTLS path
-        ConditionalPacketPath dtlsPath = new ConditionalPacketPath();
-        dtlsPath.setName("DTLS path");
-        dtlsPath.setPredicate((packet) -> {
+        Predicate<Packet> dtlsPredicate = packet -> {
+            // This comes from DTLSDatagramFilter.java, but what is the
+            // rationale?
```

Comment:
my memory was that this is the definition for how to demux dtls vs srtp packets in some rfc